surfer/viz.py

```diff
@@ -2192,7 +2196,7 @@ def save_movie(self, fname, time_dilation=4., tmin=None, tmax=None,
         fname_pattern = os.path.join(tempdir, frame_pattern)
         self.save_image_sequence(time_idx, fname_pattern, False, -1, -1,
                                  'current', interpolation=interpolation)
-        ffmpeg(fname, fname_pattern, framerate, codec)
+        ffmpeg(fname, fname_pattern, framerate, codec=codec, bitrate=bitrate)
```

Comment:
could just be `ffmpeg(fname, fname_pattern, framerate, codec, bitrate)` but that's a detail